src-v2/Not/Not.Blazor/TM/NotSimpleTable.razor

```diff
@@ -0,0 +1,45 @@
+﻿@typeparam T
+@inherits MudSimpleTable
+
+<MudSimpleTable Style="overflow-x: auto;">
+    <thead>
+        <tr>
+            @foreach (var heading in Headings)
+            {
+                <th>@heading</th>
+            }
+        </tr>
+    </thead>
+    <tbody>
+        @if(Items.Count() == 0)
+        {
+            <MudText Typo="Typo.body1">@EmptyMessage</MudText>
+        }
+        @foreach (var item in Items)
+        {
+            <tr>
+                @foreach (var cellValue in item!.ToString()!.Split("|"))
```

Comment:
The idea of `Nullable` isn't to slap `!` everywhere, but to make it more obivous where you can expect null reference errors. What happens if an Item (which isn't constrained to any type at all) returns null from ToString? While returning null isn't sensible in a ToString it's completely compile-time valid and as such should be handled gracefully.

Reply:
The new anylizers prevent compilation without these !. Not sure how to handle them more gracefully.

Reply:
I proposed a question for a reason: What happens if `ToString()` returns null?